src/avfoundation.cs

```diff
@@ -3172,6 +3172,109 @@ interface AVCaptureSynchronizedDepthData
 		AVCaptureOutputDataDroppedReason DroppedReason { get; }
 	}
 
+	[TV (11,0), NoWatch, Mac (10,13), iOS (11,0)]
+	[Protocol]
+	interface AVQueuedSampleBufferRendering
+	{
+		[Abstract]
+		[Export ("timebase", ArgumentSemantic.Retain)]
+		CMTimebase Timebase { get; }
+
+		[Abstract]
+		[Export ("enqueueSampleBuffer:")]
+		void Enqueue (CMSampleBuffer sampleBuffer);
+
+		[Abstract]
+		[Export ("flush")]
+		void Flush ();
+
+		[Abstract]
+		[Export ("readyForMoreMediaData")]
+		bool ReadyForMoreMediaData { [Bind ("isReadyForMoreMediaData")] get; }
+
+		[Abstract]
+		[Export ("requestMediaDataWhenReadyOnQueue:usingBlock:")]
+		void RequestMediaData (DispatchQueue queue, Action block);
```

Comment:
let us not use `block` as parameter name here, maybe `handler`?

Reply:
`Hander` is a valid choice, there are a lot of occurences of `completionHandler` or `completion` in our bindings but I don't think that's relevant here because of how the API works.

I would also suggest `inputHandler` based on this comment in the doc: "The block should append media data to the input" (https://developer.apple.com/documentation/avfoundation/avassetwriterinput/1387508-requestmediadatawhenreadyonqueue).